### ISSUE TITLE
[Feature Request] I can put a value on an hidden input

### ISSUE THREAD

**Issue by IgorDePaula** [vuetifyjs/vuetify#3887]
### Problem to solve
Actually, I can't write a form with hidden input

### Proposed solution
Add hide props to input component

**Comment by nekosaur** [comment]
You need to provide way more information about the current problem. I have no idea what this is about.

**Comment by IgorDePaula** [comment]
On a normal page rendered by server, I can put a hidden field with value of master on details form.

Imagine that I need build a form where I need attach many register to another one specific. I hide the ID of this specific register and show all that I can attach, I choose what I want attach and send.


My real case, I have a list of course, and I need attach disciplines in each course, I list all disciplines attached on a specific course and build a form for attach anothers disciplines. 

I show this form all disciplines that I can attach, choose which attach and send, but I send the ID course hidden (for user not choose again) and all disciplines that the user choose.

**Comment by KaelWD** [comment]
I still have no idea what you want us to do with this information.

**Comment by IgorDePaula** [comment]
The doc not inform how I can a hidden input.

**Comment by IgorDePaula** [comment]
Yes, I'm stupid for want help this wonderful tool. I already chatted and was sculpted. Thanks for your attention.
